### PR TITLE
Bump up golang version 

### PR DESCRIPTION
**Pull request by bpradipt**

### CODE REVIEW

File: ibmcloud/terraform/cluster/ansible/kata-playbook.yml [hunk 1]
```diff
@@ -57,10 +57,8 @@
       shell: |
         set -o errexit -o pipefail
         arch="{{ ansible_architecture }}"
-        # go1.18 cannot compile containerd v1.6.1. Use go1.17.8 until the issue is fixed.
-        # https://github.com/containerd/containerd/issues/6586
         # gover=$(curl -sL 'https://golang.org/VERSION?m=text')
-        gover=go1.17.8
+        gover=go1.18.5
         curl -sL "https://go.dev/dl/$gover.linux-${arch/x86_64/amd64}.tar.gz" | tar -xzf - -C /usr/local
```
Comment (yoheiueda) [inline ibmcloud/terraform/cluster/ansible/kata-playbook.yml:62]: Could you uncomment this line?
```
# gover=$(curl -sL 'https://golang.org/VERSION?m=text')
```
Then, just remove this line.
```
gover=go1.17.8
```

Could you also modify `ibmcloud/terraform/cluster/ansible/kube-playbook.yml`?

https://github.com/confidential-containers/cloud-api-adaptor/blob/5f6190e52d2625760fb70e3021f885d6481daa8b/ibmcloud/terraform/cluster/ansible/kube-playbook.yml#L130-L131

Comment (bpradipt) [reply ibmcloud/terraform/cluster/ansible/kata-playbook.yml:62]: @yoheiueda running this `curl -sL 'https://golang.org/VERSION?m=text'` returns the latest version `go-1.19`. Is it ok ?

Comment (yoheiueda) [reply ibmcloud/terraform/cluster/ansible/kata-playbook.yml:62]: OK. I rethink it. 

Let's keep using 1.18.5 until go 1.19 is stabilized. So, could you apply the same change to `kube-playbook.yml` as well?

**Review by snir911 — DISMISSED** [review]

**Review by snir911 — APPROVED** [review]
I'd squash but also fine as is, thanks!

**Pull request merged**